kpm/lib/kpm/installer.rb

```diff
@@ -39,6 +39,7 @@ def install(force_download=false, verify_sha1=true)
         unless @config['default_bundles'] == false
           install_default_bundles(@config['plugins_dir'], @config['default_bundles_version'], @config['version'], force_download, verify_sha1)
         end
+        clean_up_descriptors
       end
```

Comment:
It seems that logic will only be plugged when we install from a file. Would it make sense to also go through it when we install a specific plugin (to at least clean up entries related to that plugin)?